lms/templates/instructor/instructor_dashboard_2/e-commerce.html

```diff
@@ -146,8 +146,8 @@ <h3 class="error-msgs" id="error-msg"></h3>
 
    $(function () {
        var icons = {
-           header: "ui-fa-circle-arrow-e",
-           activeHeader: "ui-fa-circle-arrow-s"
+           header: "ui-icon-carat-1-e",
+           activeHeader: "ui-icon-carat-1-s"
```

Comment:
and @rlucioni was happy...on a unrelated note @rlucioni is no longer allowed to file bugs